js/main.js

```diff
@@ -0,0 +1,34 @@
+function getRandomNumber (min, max) {
+  return (Math.random() * (max - min)) + min;
+}
+
+function checkRange (min, max) {
+  if (min >= 0 && min < max) {
+    return true;
+  }
+
+  return false;
```

Comment:
Это нужно сократить до
```suggestion
  return min >= 0 && min < max;
```